scripts/amalgamation.py

```diff
@@ -392,6 +392,8 @@ def gather_files(dir, source_files, header_files):
         elif fname.endswith('.cpp') or fname.endswith('.c') or fname.endswith('.cc'):
             gather_file(fpath, source_files, header_files)
 
+def write_license(hfile):
+    hfile.write("// See https://raw.githubusercontent.com/cwida/duckdb/master/LICENSE for licensing information\n\n")
```

Comment:
probably worth updating the URL while we're at it?

Reply:
Sure, I just copied it from the old version of this function that was removed. What URL should I use?

Reply:
The new URL is duckdb/duckdb, not cwida/duckdb, although the latter redirects to the former